installation-bundle/src/Controller/InstallationController.php

```diff
@@ -567,7 +567,7 @@ private function render(string $name, array $context = []): Response
 
     private function trans(string $key): string
     {
-        return $this->container->get('translator')->trans($key);
+        return $this->container->get('translator')->trans($key, [], 'ContaoInstallationBundle');
```

Comment:
shouldn't this be like `contao_installation` or the like?

Reply:
No, the Symfony best practices suggest to use the bundle name as message domain (see 5d5e6e089ca3936ba443ab305e20ab9778b1a11c).

Reply:
Ok, didn't know that 🙃